moveit_benchmark_resources/package.xml

```diff
@@ -4,7 +4,7 @@
   <version>1.0.0</version>
   <description>Databases for MoveIt benchmarking</description>
 
-  <author email="sebastian.jahr@picknik.ai">Ioan Sucan</author>
+  <author email="sebastian.jahr@picknik.ai">Sebastian Jahr</author>
```

Comment:
```suggestion
  <author email="sebastian.jahr@picknik.ai">Sebastian Jahr</author>
  <author email="cihataltiparmak1@gmail.com">Cihat Altıparmak</author>
```

Reply:
Ohh my god, thank you so much, but is it really necessary?

Reply:
If you don't mind :+1: The bag files will probably make up half of what is in this package so I think we can add you here

Reply:
Thank you:)